test/mocha.opts

```diff
@@ -1,3 +1,4 @@
 --ui tdd
 --timeout 240s
 --reporter spec
+--exit
```

Comment:
I think our next spring cleaning needs to focus on tests, and we can get rid of this :)